dev/bots/analyze-sample-code.dart

```diff
@@ -99,15 +99,21 @@ class Section {
 const String kDartDocPrefix = '///';
 const String kDartDocPrefixWithSpace = '$kDartDocPrefix ';
 
-Future<Null> main() async {
+Future<Null> main(List<String> arguments) async {
   final Directory temp = Directory.systemTemp.createTempSync('analyze_sample_code_');
   int exitCode = 1;
   bool keepMain = false;
   final List<String> buffer = <String>[];
   try {
     final File mainDart = new File(path.join(temp.path, 'main.dart'));
     final File pubSpec = new File(path.join(temp.path, 'pubspec.yaml'));
-    final Directory flutterPackage = new Directory(path.join(_flutterRoot, 'packages', 'flutter', 'lib'));
+    Directory flutterPackage;
+    if (arguments.length == 1) {
```

Comment:
How about using the args package here? Or at least do something simple like force it to be length 2, where the first arg has to be "--package".

That way, if we add more arguments someday, this one will be named, and not taking up the unnamed "first argument" spot, and the callers (probably) won't have to change how they call it.